purchase_requisition_from_rfq/models/purchase.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.multi
+    def write(self, vals):
+        for rec in self:
+            super(PurchaseOrder, rec).write(vals)
+
+    @api.multi
+    def button_confirm(self):
+        res = super(PurchaseOrder, self).button_confirm()
+        PR = self.env['purchase.requisition']
```

Comment:
Let's don't use the word "PR" it is confused with "purchase request". Use the word "Requisition"

Reply:
Changed